code/modules/mining/equipment/kinetic_crusher.dm

```diff
@@ -443,3 +461,117 @@
 
 /obj/effect/temp_visual/hierophant/wall/crusher
 	duration = 75
+
+//Legion(The huge one)
+
+/obj/item/crusher_trophy/legion_shard
+	name = "legion shard"
+	desc = "A still moving piece of legion's skull... Looks like it can be attacked to a kinetic crusher."
+	icon_state = "legion_shard"
+	denied_type = /obj/item/crusher_trophy/legion_shard
+	bonus_value = 17
+
+/obj/item/crusher_trophy/legion_shard/effect_desc()
+	return "mark detonation to have a chance of spawning a friendly legion skull"
+
+/obj/item/crusher_trophy/legion_shard/on_mark_detonation(mob/living/target, mob/living/user)
+	if(prob(15))
+		var/mob/living/simple_animal/hostile/asteroid/hivelordbrood/legion/A = new /mob/living/simple_animal/hostile/asteroid/hivelordbrood/legion(user.loc)
+		A.GiveTarget(target)
+		A.friends[user]++
+		A.faction = user.faction.Copy()
+
+//Abyss Demon
+
+/obj/item/crusher_trophy/abyssal_crystal
+	name = "abyssal crystal"
+	desc = "A red, glowing crystal that was sliced off the Abyssal Demon. Suitable as a trophy for a kinetic crusher"
+	icon_state = "crystal_shard"
+	denied_type = /obj/item/crusher_trophy/abyssal_crystal
+	bonus_value = 5
+
+/obj/item/crusher_trophy/abyssal_crystal/effect_desc()
+	return "mark detonation to pierce target with crystal shards and stun him for [bonus_value * 0.1] seconds"
+
+/obj/effect/temp_visual/abyss_shards
+	name = "abyssal shards"
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "bloody_trap"
+	duration = 10
+	layer = BELOW_MOB_LAYER
+
+/obj/item/crusher_trophy/abyssal_crystal/on_mark_detonation(mob/living/target, mob/living/user)
+	new /obj/effect/temp_visual/abyss_shards(get_turf(target))
+	target.Stun(bonus_value * 0.1)
```

Comment:
this won't do anything to anything but carbons, who marks cant be detonated on- unless, god forbid, you changed that fact

Reply:
1. You can't detonate marks on humans
2. You CAN stun animals.
/mob/living/proc/AdjustStun(amount, updating = TRUE, ignore_canstun = FALSE) //Adds to remaining duration